Composer/packages/client/src/store/action/export.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ActionCreator } from '../types';
+
+import httpClient from './../../utils/httpUtil';
+import { setError } from './error';
+
+export const exportToZip: ActionCreator = async (store, { projectId }) => {
```

Comment:
It would probably be useful to add a comment here explaining that this is the only way to force all browsers to download the file instead of trying to open it.